src/bitcoin-cli.cpp

```diff
@@ -85,7 +85,7 @@ static bool AppInitRPC(int argc, char* argv[])
         }
 
         fprintf(stdout, "%s", strUsage.c_str());
-        return false;
+        if(argc < 2) return false; else exit(0);
```

Comment:
I don't like that kind of code mix either but returning true here instead of exiting early fails with `too few parameters (need at least command)` at `CommandLineRPC` later.
```
> ./src/bitcoin-cli --version; echo $?
Bitcoin Core RPC client version v0.13.99.0-e6d8192
error: too few parameters (need at least command)
1
```